websites/K/KaryaKarsa/presence.ts

```diff
@@ -0,0 +1,231 @@
+const presence = new Presence({
+    clientId: "920135248006754394"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000),
+  pathData = {
+    user: [
+      {
+        name: "supports",
+        state: "My Subscriptions History"
+      },
+      {
+        name: "edit",
+        state: "Editing my details"
+      }
+    ],
+    dashboard: [
+      {
+        name: "overview",
+        state: "My Overview"
+      },
+      {
+        name: "onboarding",
+        state: "Starting Up"
+      },
+      {
+        name: "goal",
+        state: "My Goal"
+      },
+      {
+        name: "tiers",
+        paths: [
+          {
+            name: "",
+            state: "My Support Tiers"
+          },
+          {
+            name: "setup",
+            state: "Setting up Supporter Tiers..."
+          },
+          {
+            name: "add",
+            state: "Adding new Supporter Tier..."
+          },
+          {
+            name: "*",
+            state: "Editing Supporter Tier..."
+          }
+        ]
+      },
+      {
+        name: "posts",
+        paths: [
+          {
+            name: "",
+            state: "My Posts"
+          },
+          {
+            name: "add",
+            state: "Making new post..."
+          },
+          {
+            name: "*",
+            state: "Editing my post..."
+          }
+        ]
+      },
+      {
+        name: "vouchers",
+        paths: [
+          {
+            name: "",
+            state: "My Vouchers"
+          },
+          {
+            name: "add",
+            state: "Making new Voucher..."
+          },
+          {
+            name: "*",
+            state: "Editing my Voucher..."
+          }
+        ]
+      },
+      {
+        name: "overlay",
+        state: "Stream Overlay"
+      },
+      {
+        name: "supports",
+        state: "Supports History"
+      },
+      {
+        name: "bank",
+        state: "Payment"
+      },
+      {
+        name: "profile",
+        state: "Editing Profile"
+      },
+      {
+        name: "settings",
+        state: "Settings"
+      }
+    ]
+  },
+  getUserHeader = ".q-page-container.q-pb-xl.main",
+  getUserHeaderHome =
+    ".title.q-toolbar__title.ellipsis a.no-underline.text-primary.router-link-active",
+  // This function act like switch function but for objects Array (yea its a lil bit complicated)
+  stateSetter = (
+    paths: string[],
+    arrayPath: Array<{
+      name: string;
+      state?: string;
+      paths?: Array<{
+        name?: string;
+        state?: string;
+      }>;
+    }>
+  ): string => {
+    const result = arrayPath.find((p) => p.name === (paths[1] || ""));
+    if (result.state) return result.state;
+    return result.paths.find((p) => {
+      const b: boolean = p.name === (paths[2] || "");
+      if (!b && !!paths[2]) return p.name === "*";
+      return b;
+    }).state;
```

Comment:
Use nullish coalescing and avoid unnecessary type assertions
```suggestion
    const result = arrayPath.find((p) => p.name === (paths[1] ?? ""));
    return result.state ?? result.paths.find(p => {
      const b = p.name === (paths[2] ?? "");
      if (!b && paths[2]) return p.name === "*";
      return b;
    }).state;
```